README.rst

```diff
@@ -126,6 +126,10 @@ To add the default dashboards to the new user, add the ``--bootstrap-dashboards`
 
     tutor local do cairn-createuser --course-id='course-v1:edX+DemoX+Demo_Course' YOURUSERNAME YOURUSERNAME@YOUREMAIL.COM
 
+By default, AUTH_ROLES_SYNC_AT_LOGIN is False which means admin can customize the permissions associated to a user. To disable this behaviour, modify the ``CAIRN_AUTH_ROLES_SYNC_AT_LOGIN`` setting::
+
+    tutor config save --set CAIRN_AUTH_ROLES_SYNC_AT_LOGIN=True
```

Comment:
nit.: It should be lowercase "true", because config settings are stored as yaml. I mean, "True" also seems to work, but I have no idea why :sweat_smile: